tests/cross-language/go/const.go

```diff
@@ -0,0 +1,25 @@
+package cltf
+
+import (
+	"fmt"
+	"os"
+)
+
+const (
+	KeyManagementStaticMasterKey = "mysupersecretstaticmasterkey!!!!"
+
+	DefaultServiceID   = "service"
+	DefaultProductID   = "product"
+	DefaultPartitionID = "partition"
+
+	FileDirectory = "/tmp/"
+	FileName      = "go_encrypted"
```

Comment:
As we're only dealing with a single package here there's no need to export any of these